efficientdet/keras/train.py

```diff
@@ -214,7 +215,7 @@ def get_dataset(is_training, params):
 
     if FLAGS.pretrained_ckpt:
       ckpt_path = tf.train.latest_checkpoint(FLAGS.pretrained_ckpt)
-      model.load_weights(ckpt_path)
+      util_keras.restore_ckpt(model, ckpt_path, params['moving_average_decay'])
```

Comment:
Try to load ckpt via restore_ckpt which is compatible with tf1.x ckpt and support load ema variables.

Reply:
great, thanks!